app/js/fm-player/config.js

```diff
@@ -28,6 +28,12 @@ angular.module("sn.fm.player").config([
                     }],
                     currentTrack: ["PlayerTransportResource", function (PlayerTransportResource){
                         return PlayerTransportResource.get();
+                    }],
+                    PlayerVolumeResource: ["PlayerVolumeResource", function (PlayerVolumeResource){
+                        return PlayerVolumeResource.get();
+                    }],
+                    PlayerMuteResource: ["PlayerMuteResource", function (PlayerMuteResource){
+                        return PlayerMuteResource.get();
                     }]
```

Comment:
Originally I was getting the volume data so we could set the position of the slider, with the buttons I don't think we would need it. Although we may still need to get the mute state so we can display it.